src/rules/MD/MDRules.js

```diff
@@ -0,0 +1,122 @@
+const util = require("../../util");
+const fs = require("fs");
+const os = require('os');
+
+module.exports = {
+  /**
+   * Analyzes whether the first section of the file is of the determined order.
+   * 
+   * @param {*} link File path.
+   * @param {*} rules Rules file.
+   */
+  ruleFirstSectionStartsWithHx: function (link, rules) {
```

Comment:
I believe that Markdown rules should receive the context object in their rules
```suggestion
ruleFirstSectionStartsWithHx: function (link, rules, context) {
```